matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/Session.kt

```diff
@@ -248,6 +249,11 @@ interface Session {
      */
     fun permalinkService(): PermalinkService
 
+    /**
+     * Returns the deferredPermalinkService service associated with the session.
+     */
+    fun deferredPermalinkService(): DeferredPermalinkService
```

Comment:
I am not sure to understand why this code is on the SDK side. It should be at the application level.